spec/instance_solr_doc_spec.rb

```diff
@@ -39,6 +39,14 @@
       allow(isd).to receive(:instance_uri_to_ckey).and_return('666')
       expect(doc_hash[:format_main_ssim]).to eq 'Book'
     end
+    it 'assigns DOC_SOURCE to access_facet' do
+      expect(doc_hash.keys).to include(:access_facet)
+      expect(doc_hash[:access_facet]).to eq(SparqlToSwSolr::InstanceSolrDoc::DOC_SOURCE)
+    end
+    it 'assigns DOC_SOURCE to collection' do
+      expect(doc_hash.keys).to include(:collection)
+      expect(doc_hash[:collection]).to eq(SparqlToSwSolr::InstanceSolrDoc::DOC_SOURCE)
+    end
```

Comment:
given the refactoring to `init_doc` method, I would move the appropriate tests into a context block for `.init_doc`